Projects/LES01-AWS100.md

```diff
@@ -0,0 +1,44 @@
+
+# LES01-AWS100 - Create a Serverless API 
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+[Syed Auther](https://twitter.com/syedauther)
+
+## Objectives
+
+### You need to complete the following:
+- Create an AWS Lambda with a language of your choice, the purpose of this Lambda is to respond with a 'Hello Serverless World!' message.
+- Create an API Gateway endpoint and connect it to the AWS Lambda function created above and deploy it as an API that can be consumed from the Browser/ Postman.
+- Secure the API with an API Key
+
```

Comment:
Nice and concise!

Reply:
"Secure the API with an API Key" definitely not Level 100 IMO. I would not add API Keys or Authorizers with a first APIGW/Lambda project, especially not on Level 100.